src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunctionImplementation.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.PowerFx.Core.Utils;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    internal class TexlFunctionImplementation : ITexlFunction
```

Comment:
>TexlFunctionImplementation [](http://example.com/codeflow?start=19&length=26)

Comment about what this is?

Reply:
Will add

Reply:
Also not placed in the right project, I moved it to Interpreter